packages/plugin-ext-vscode/src/node/plugin-vscode-init-fe.ts

```diff
@@ -0,0 +1,43 @@
+/********************************************************************************
+ * Copyright (C) 2022 Arm
```

Comment:
Nitpick: I believe we usually add `and others.` even if the file hasn't been touched by others.
```suggestion
 * Copyright (C) 2022 Arm and others.
```